custom_components/oncharger/config_flow.py

```diff
@@ -60,6 +63,7 @@
 USER_SCHEMA = vol.Schema(
     {
         vol.Required(CONNECTION_TYPE, default=LOCAL): vol.In((LOCAL, CLOUD)),
+        vol.Required(DEVICE_TYPE, default=SINGLE_PHASE): vol.selector((SINGLE_PHASE, THREE_PHASE)),
```

Comment:
Let's see if we can make this detection automatic

Reply:
We can't detect it reliably and 3p charger can work as a 1p charger as well so config option is better